docs/notebooks/PulsediveLookup.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
Any chance that you can create an RST doc from the notebook and add to docs/source/data_acquisition
You will need to tidy it up a bit, but you can generate an RST using nbconvert:
```
jupyter nbcovert --to rst your_notebook.ipynb

```
Name it something like PulseDive.rst
Add a line to link it from docs/source/DataEnrichment.rst
```rst
.. toctree::
   :maxdepth: 2

   data_acquisition/TIProviders
   data_acquisition/GeoIPLookups
   data_acquisition/AzureData
   data_acquisition/Sentinel
   data_acquisition/IPWhois
   data_acquisition/PulseDive
```

To generate the docs:
```
cd docs
make html nogen
# open docs/build/html/index
```
You should be able to see your doc in the ToC under the Enriching section
You should also be able to find the module in the API section.
(although for this to be generated you'll need to do `make html`  - omitting the "nogen")